src/index.js

```diff
@@ -1,16 +1,24 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { createStore } from 'redux';
+import { createStore, compose, applyMiddleware } from 'redux';
 import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
 import registerServiceWorker from './registerServiceWorker';
 
 import './index.css';
 import App from './App';
 
-
 import rootReducer from './reducers';
+// import updateServer from './middleware/upadteServer';
```

Comment:
??

Reply:
This is my mistake, sorry, I know that I can't push commented code. I was learning middleware, and this line stayed.

Reply:
Ok. I guess it's removed in the following PR's